replication/backup.go

```diff
@@ -67,26 +95,26 @@ func (b *BinlogSyncer) StartBackup(backupDir string, p Position, timeout time.Du
 		} else if e.Header.EventType == FORMAT_DESCRIPTION_EVENT {
 			// FormateDescriptionEvent is the first event in binlog, we will close old one and create a new
 
-			if f != nil {
-				f.Close()
+			if w != nil {
+				_ = w.Close()
```

Comment:
I think the error of `Close` is important, for example, some implementations only flush the data when `Close`. In other words, `Write` may only store data in memory and can not guarantee the persistence. 

also for line 68